backend/src/main/java/com/epam/todo/repository/ToDoRepository.java

```diff
@@ -0,0 +1,20 @@
+package com.epam.todo.repository;
```

Comment:
NOTE: You don't need two repository class. Use the hibernate one. If it's neccesary, you can add this repository class later on. :)

Reply:
Done.